rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RestartDownNodesPolicy.java

```diff
@@ -47,73 +46,93 @@
 
     private static Logger logger = Logger.getLogger(RestartDownNodesPolicy.class);
 
-    private Timer timer = new Timer("RestartDownNodesPolicy node status check");
+    private static final String TIMER_NAME = "RestartDownNodesPolicy node status check";
 
-    @Configurable(description = "ms (30 mins by default)")
+    @Configurable(description = "ms (30 mins by default)", dynamic = true)
     private int checkNodeStateEach = 30 * 60 * 1000; // 30 mins by default;
 
+    private Timer timer;
+
     /**
      * Configure a policy with given parameters.
      * @param policyParameters parameters defined by user
      */
     @Override
     public BooleanWrapper configure(Object... policyParameters) {
         BooleanWrapper parentConfigured = super.configure(policyParameters);
+        setCheckNodeStateEach(policyParameters);
+        this.timer = new Timer(TIMER_NAME);
+        return parentConfigured;
+    }
+
+    @Override
+    public void reconfigure(Object... updatedPolicyParameters) {
```

Comment:
even if it is empty for now, you should call super.reconfigure

Reply:
Yes done.